pkgs/applications/blockchains/polkadot/default.nix

```diff
@@ -61,9 +61,7 @@ rustPlatform.buildRustPackage rec {
     rustc-wasm32.llvmPackages.lld
   ];
 
-  buildInputs = [
-    rust-jemalloc-sys-unprefixed
```

Comment:
This was added in https://github.com/NixOS/nixpkgs/pull/243324 but it's not needed since by default jemalloc isn't used unless the features flag `jemalloc-allocator` is enabled (https://github.com/paritytech/polkadot-sdk/blob/master/polkadot/Cargo.toml#L61-L66).

Reply:
Should this be a top-level option?

Reply:
```
❯ ldd /nix/store/mv9k5i0cmz41dz7qfsfvsq6zy58y4mhb-polkadot-1.2.0/bin/polkadot | grep -i jemalloc
	libjemalloc.so.2 => /nix/store/h2f8fvdjdh3wq2qq68134705j31sc2p0-jemalloc-5.3.0/lib/libjemalloc.so.2 (0x00007fd155400000)
```

Works as expected.

Reply:
OK, and should there be some mention of why it uses the `-unprefixed` version? Or not necessary?

Reply:
This wasn't explicitly enabled in polkadot but was indirectly enabled by a dependency on the rocksdb crate (the way cargo works it unifies all features for the same dependency). I made a PR to polkadot to explicitly enable this and make it more obvious (https://github.com/paritytech/polkadot-sdk/pull/2002).

Reply:
I updated the comment here to say that unprefixed variant is used in polkadot. It also fails to build if you try to use the prefixed version so not a lot a room for mistakes.